packages/itwinui-react/setupTests.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import '@testing-library/jest-dom';
+import '@testing-library/jest-dom/vitest';
 import { TextEncoder } from 'util';
```

Comment:
`TextEncoder` might not be necessary to mock anymore. I believe it was added to work around some jest-specific issue